RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py

```diff
@@ -32,6 +32,10 @@
     Phase2KinkADC = cms.int32(8),
 )
 
+# *only for the cms-patatrack repository*
+# ensure reproducibility for CPU <--> GPU comparisons
+siPixelClusters.payloadType = "HLT"
```

Comment:
is this really meant for CMSSW?

Reply:
No, that's one of the changes we need to revert (or maybe make conditional on the GPU-only workflows, for the time being).
Noted in https://github.com/cms-patatrack/cmssw/issues/200 .